core/soc/estimator.go

```diff
@@ -27,6 +27,9 @@ type Estimator struct {
 	prevSoc           float64 // previous vehicle Soc in %
 	prevChargedEnergy float64 // previous charged energy in Wh
 	energyPerSocStep  float64 // Energy per Soc percent in Wh
+	minChargePower    float64 // Lowest charge power (before vehicle stops charging)
+	maxChargePower    float64 // Highest charge power the battery can handle on any charger
+	maxChargeSoc      float64 // SoC at/after which maxChargePower is reduced
```

Comment:
max(Charge)PowerSoc

Generell- das `Charge` in den Bezeichnern könnte weg? Andere Leistungen gibt es ja im estimator nicht?

Reply:
Weiss nicht. Es solle ja ein Zusammenhang der beiden Punkte im Ladekurvenkoordinatensystem hergestellt werden die nichts mit der sonstigen Terminologie wie z. B. minSoc, targetSoc in evcc zu tun haben. Fand das so erstmal weniger verwirrend.